crates/polars-core/src/chunked_array/ops/lambda.rs

```diff
@@ -140,10 +140,95 @@ impl LambdaExpression {
                 AnyValue::List(series)
             },
             LambdaExpression::GreaterThan(left, right) => {
-                left.eval_array(args).gt(&right.eval_array(args)).into()
+                // CURRENTLY ONLY SUPPORTS NESTED ARRAYS THAT HAVE A SINGLE MEMBER.
```

Comment:
Honestly, approved but this entire mechanism has greatly overstayed its usefulness, It was meant to be a quick solution, we need to redesign this entire thing

Reply:
Yeah I understand what you mean, it'll be an interesting discussion to see how we can make this robust.